angular-app/src/app/app.module.ts

```diff
@@ -100,6 +101,13 @@ import { ProjectService } from './project.service';
   providers: [
     AuthService,
     ProjectService,
+    {
+      provide: APP_INITIALIZER,
+      useFactory: (projectService: ProjectService) =>
+      () => projectService.load(),
+      deps: [ProjectService],
+      multi: true,
```

Comment:
Hmm... It doesn't seem like this should be necessary for the loading icon to work properly. But if this works for now, that is fine.